flipt-client-kotlin-android/src/androidTest/java/io/flipt/client/TestFliptEvaluationClient.kt

```diff
@@ -15,14 +14,16 @@ class TestFliptEvaluationClient {
     fun initAll() {
         val fliptURL = BuildConfig.FLIPT_URL
         val clientToken = BuildConfig.FLIPT_AUTH_TOKEN
-
+        assert("http://10.0.2.2:8080" == fliptURL)
```

Comment:
so Flipt should be running on localhost now right? since its running in the action. Does the emulator give localhost a different IP?

Reply:
Android emulator is a vm. Similar to docker there are reserved IPs for accessing something in the network or host https://developer.android.com/studio/run/emulator-networking . I debugged it locally with `adb logcat` and everything looks like with ffi options.

For some reason tests could not access network. It doens't matter that the host is. try.flipt.io could not be reach as well. It must be something with permissions or network security policies.